mmv1/products/alloydb/Cluster.yaml

```
@@ -559,3 +559,37 @@ properties:
                   name: nanos
                   description: |
                     Fractions of seconds in nanoseconds. Currently, only the value 0 is supported.
+  - !ruby/object:Api::Type::Enum
+    name: 'subscriptionType'
+    values:
+      - :TRIAL
+      - :STANDARD
+    default_from_api: true
+    description: |
+      The subscrition type of cluster.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'trialMetadata'
+    description: |
+      Contains information and all metadata related to TRIAL clusters.
+    default_from_api: true
```

```suggestion
    output: true
```

Similar change needed for all sub-fields here if this is output-only

---

Not all these fields are always populated. For example, trialMetadata.UpgradeTime is populated only if a cluster is upgraded from trial to standard. And, trialMetadata.StartTime is populated only for a trial cluster, not for a standard cluster. These fields are null otherwise. Should all the sub-fields needed to be marked as output-only individually?